### DISCUSSION TITLE
is your private key meant to be pushed to heroku?

### DISCUSSION THREAD

**Discussion by benkoshy** [thekompanee/chamber#80] [Q&A] [answered]
When you push some keys to heroku, what keys are actually saved?

Is the private key also pushed to heroku?

**Comment by jfelchner** [comment, marked as answer]
Yes, your private key is meant to be pushed to Heroku.  Heroku is considered to be a secure party in this process.  It has access to your running code, your DB password in plain text, etc.  So pushing your private chamber key to Heroku is what you want.
